features/supplier/brief_response.feature

```diff
@@ -147,3 +147,94 @@ Scenario: Supplier applies for a user-research-participants brief
       | Liking sugar lumps                 |                      |
       | Being good at jumping over fences  | No jump is too high. |
       | Saying "Neigh"                     | NEIGH                |
+
+Scenario: Previous page links are used during response flow
```

Comment:
Sorry, to make myself clearer, I realise that this scenario is asserting those things but I mean to include this in the _description_ of the scenario e.g.
Previous page links are used and existing data prefilled during response flow